cpp/src/com/google/idea/blaze/cpp/BlazeConfigurationResolver.java

```diff
@@ -133,12 +133,12 @@ private static ImmutableMap<String, String> getTargetToVersionMap(ImmutableMap<T
     ImmutableMap<ExecutionRootPath, String> compilerVersionByPath =
             compilerSettings.entrySet().stream().collect(
                     ImmutableMap.toImmutableMap(
-                            e -> e.getKey().getCppExecutable(),
-                            e -> e.getValue().getCompilerVersion()));
+                            e -> e.getKey().getCppCompiler(),
+                            e -> e.getValue().getCppCompilerVersion()));
```

Comment:
This not correct and I already had a simple fix for `BlazeCompilerInfoMapService`. But maybe it is worth taking a closer look considering #6410. Since this would require more changes I would like to merge this first.